### ISSUE TITLE
No badge notifications

### ISSUE THREAD

**Issue by pawliktamas** [inbasic/ignotifier#488]
Hi,

No badge notifications :(
There are never badge maths!

The last good version, 0.7.4. No next version is unable to display badges.
I use it on iMac, MacOS Sierra, with the latest version of FireFox 57.0.

Please help me!
Thanks in advance!

**Comment by pawliktamas** [comment]
Hi,

No badge notifications :(
There are never badge maths!

The last good version, 0.7.4. No next version is unable to display badges.
I use it on iMac, MacOS Sierra, with the latest version of FireFox 57.0.

Please help me!
Thanks in advance! # @inbasic

**Comment by inbasic** [comment]
Please switch to 0.8.2. You have a very old version!
https://addons.mozilla.org/en-US/firefox/addon/gmail-notifier-restartless/

**Comment by pawliktamas** [comment]
@inbasic 
Thanks for the reply!
I already have Version 0.8.2, but there are no badges.
The last good version, 0.7.4. When I returned this old version, it worked perfectly. Since then, the new version does not work with the badge.
What could be the problem?
Please help.

**Comment by inbasic** [comment]
@pawliktamas can you confirm this on a clean profile? Also make sure third Party cookies is not being blocked by any legacy extension.

**Comment by pawliktamas** [comment]
@inbasic 
What does it mean: "can you confirm this on a clean profile"?

My other extension is just above:
https://www.basson.at/firefox-addons/newtabhomepage

I have not changed anything since the release of the new version. Unfortunately this bug was brought by the new version :(

Thanks for your help, waiting for you!

**Comment by inbasic** [comment]
This is pretty simple to create a new clean profile. Make a temporary one and see if Gmail Notifier works there or not; https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles

**Comment by pawliktamas** [comment]
@inbasic 
I have created a new clean profile, but unfortunately this is the same problem. No badge is coming :(

**Comment by inbasic** [comment]
how about 0.8.4 release?

**Comment by pawliktamas** [comment]
@inbasic 
I did get the latest version, but unfortunately it was not good :(
I give you two pictures of where the badge is missing.
![badge yes](https://user-images.githubusercontent.com/33716048/33852797-18913934-debc-11e7-839b-9ba04b27ba7d.jpg)
![badge not](https://user-images.githubusercontent.com/33716048/33852802-1b409d00-debc-11e7-8dbe-1e205baf0f75.jpg)

**Comment by inbasic** [comment]
There is no way to have this 3 any more! There is no WebExtension API for that!

**Comment by inbasic** [comment]
duplicate of https://github.com/inbasic/ignotifier/issues/482
